### DISCUSSION TITLE
Where can I set --save_last_n_epochs_state in the gui?

### DISCUSSION THREAD

**Discussion by Vigilence** [bmaltais/kohya_ss#2909] [Q&A] [answered]
I can't seem to find it.

**Comment by b-fission** [comment]
The gui doesn't have a dedicated setting for that, but you can specify it in Advanced->Additional parameters.
Pasting in `--save_last_n_epochs_state 1` should do it.

**Comment by Vigilence** [comment]
Ok great, thanks so much!  Hopefully @bmaltais can add it to the gui in the future.

**Comment by bmaltais** [comment]
Let me check why it is not there...

**Comment by bmaltais** [comment, marked as answer]
Pushed support in the 'sd3-flux.1' branch.

Look under the Advanced section with the save_last_n_steps_state

**Reply by Vigilence** [reply]
Sweet ty!
